totalopenstation/tests/test_xml.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+
+from totalopenstation.formats import Feature, Point
+from totalopenstation.output.tops_xml import OutputFormat
+
+class TestXMLOutput(unittest.TestCase):
```

Comment:
Can we have also a `TestLandXMLParser` so both input and output are covered?

Reply:
Done